haskell/private/haskell_impl.bzl

```diff
@@ -308,6 +308,7 @@ def haskell_library_impl(ctx):
         other_modules,
         my_pkg_id,
         static_library_prof = static_library_prof,
+        short_path = ctx.executable._short_path if hs.toolchain.is_windows else None,
```

Comment:
I’d call this `executable_short_path`.

Reply:
What do you think of `shorten_path` instead ? `executable_short_path` does not carry the same meaning to me. It sounds like this is really the short path of some executable, while this is instead an executable (script) used to shorten the path of (in this case) a directory.

Reply:
I'll go with `short_path_executable`.